src/figures/figureB1.jl

```diff
@@ -1,41 +1,28 @@
 """ This file builds the depletion manuscript, Figure 1. """
 
 """ Plot an example isobologram. """
-function plotIsobologram(IgGXidx::Int64, IgGYidx::Int64; murine = false, c1q = false)
+function plotCellIsobologram(IgGXidx::Int64, IgGYidx::Int64, Cellidx::Int64; L0 = 1e-9, f = 4, murine = true, c1q = false, ex = false)
+    CellName = ["ncMO", "cMO", "NKs", "Neu", "EO"]
+    Cell = CellName[Cellidx]
     Xname = murine ? murineIgG[IgGXidx] : humanIgG[IgGXidx]
     Yname = murine ? murineIgG[IgGYidx] : humanIgG[IgGYidx]
-    Kav = importKav(murine = murine)
-    # TODO: Should import actual receptor abundance
-    FcExpr = zeros(length(humanFcgR))
-    FcExpr[7] = importRtot(murine = murine)[7, 2]
-
-    output = calculateIsobologram(IgGXidx, IgGYidx, 24, 1.0e-8, FcExpr, Kav)
-
-    X = range(0, stop = 1, length = length(output))
-
-    pl = plot(
-        layer(x = X, y = output, Geom.line, Theme(default_color = colorant"green")),
-        layer(x = [0, 1], y = [output[1], output[end]], Geom.line, Theme(default_color = colorant"red")),
-        Scale.x_continuous(labels = n -> "$Xname $(n*100)%\n$Yname $(100-n*100)%"),
-        Scale.y_continuous(minvalue = 0.0, maxvalue = 1.0),
-        Guide.xlabel("Percent hIgG3"),
-        Guide.ylabel("hFcgRIIIA-158V Binding"),
-        Guide.manual_color_key("", ["Predicted", "Linear Addition"], ["green", "red"]),
-        Guide.title("Receptor Binding vs IgG Composition"),
-        Theme(key_position = :inside),
-    )
-    return pl
-end
-
-""" Plot an example isobologram. """
-function plotIsobologramTwo(IgGXidx::Int64, IgGYidx::Int64; murine = true, c1q = false)
-    Xname = murine ? murineIgG[IgGXidx] : humanIgG[IgGXidx]
-    Yname = murine ? murineIgG[IgGYidx] : humanIgG[IgGYidx]
-    Kav = importKav(murine = murine, IgG2bFucose = true)
-    FcExpr = importRtot(murine = murine)[:, 2]
+    Kav_df = importKav(; murine = murine, c1q = c1q, retdf = true)
+    Kav = Matrix{Float64}(Kav_df[!, murine ? murineFcgR : humanFcgR])
+    ActI = murine ? murineActI : humanActI
+    FcExpr = importRtot(murine = murine)[:, Cellidx]
+    if ex
+        FcExpr = zeros(length(humanFcgR))
+        FcExpr[7] = importRtot(murine = murine)[7, Cellidx]
+        ActI = nothing
+    end
 
-    output = calculateIsobologram(IgGXidx, IgGYidx, 4, 1.0e-9, FcExpr, Kav, actV = murineActI)
-    output /= maximum(output)
+    output = calculateIsobologram(IgGXidx, IgGYidx, f, L0, FcExpr, Kav, actV = ActI)
+    if ex
+        title = "Binding"
+    else
+        output /= maximum(output) + eps()
```

Comment:
@stephendl102 

All the activities were 0.0, so you were getting a divide by 0.0 here, turning the values into NaN. Should work now.